core-services/template-deprecation/_allowlist.yaml

```diff
@@ -763,6 +763,9 @@ templates:
         pull-ci-openshift-installer-release-4.8-e2e-aws-rhel8:
           generated: false
           kind: presubmit
+        pull-ci-openshift-machine-config-operator-master-e2e-aws-single-node:
```

Comment:
No. Here, you are adding this new job with an "unknown" blocker. Please either:

 - clearly document the blocker for migrating this type of job to multi-stage steps
 - write a multi-stage job

Reply:
xref: https://docs.ci.openshift.org/docs/how-tos/migrating-template-jobs-to-multistage/#openshift-installer

Reply:
- write a multi-stage job

Went with this option eventually. Please review #15187 so I can modify this current PR to use multi-stage.